Sources/YBottomSheet/BottomSheetController.swift

```diff
@@ -149,6 +149,14 @@ public class BottomSheetController: UIViewController {
         didDismiss()
         return true
     }
+    
+    /// Dismiss bottom sheet.
+    /// - Parameter isCloseButton: whether close button is tapped or not.
+    func didDismiss(isCloseButton: Bool = false) {
+        if appearance.allowDismiss || isCloseButton {
+            onDismiss()
+        }
+    }
```

Comment:
```suggestion
    /// Dismisses the bottom sheet if allowed.
    ///
    /// This method is not called when the header's close button is tapped.
    /// - Parameter isCloseButton: whether close button is tapped or not.
    func didDismiss() {
        if appearance.allowDismiss {
            onDismiss()
        }
    }
```
I think I would rather do this and then have
```
extension BottomSheetController: SheetHeaderViewDelegate {
    @objc
    func didCloseTapped() {
        // Directly dismiss the sheet without considering `isDismissAllowed`.
        onDismiss()
    }
}
```